yarn-project/circuits.js/src/structs/private_circuit_public_inputs.test.ts

```diff
@@ -31,4 +31,10 @@ describe('PrivateCircuitPublicInputs', () => {
     const fields = inputs.toFields();
     expect(fields.length).toBe(PRIVATE_CIRCUIT_PUBLIC_INPUTS_LENGTH);
   });
+
+  it('hash matches snapshot', () => {
+    const target = makePrivateCircuitPublicInputs(327);
```

Comment:
Why 327? Have seen you use it multiple times now?

Reply:
It's just a random fixed number I put there and I copied the same test multiple times.